lib/std/c.zig

```diff
@@ -121,10 +121,15 @@ pub fn getErrno(rc: anytype) c.E {
 
 pub extern "c" var environ: [*:null]?[*:0]u8;
 
+pub const O_RDONLY: c_uint = 0x00;
+pub const O_WRONLY: c_uint = 0x01;
+pub const O_RDWR: c_uint = 0x02;
```

Comment:
these and the others should be in a struct like so https://github.com/ziglang/zig/blob/master/lib/std/os/linux.zig#L144

Reply:
Yes but they're not the values found in the libc directly. I thought duplicating them in ``std/c.zig`` with the original name made sense.

Reply:
they're not being `extern`'d so that's not as much of a concern. the values have to match the ones in `std.os` because `std.c` becomes `std.os.system` when libc is linked in a build

Reply:
I see, but the thing is that it's in ``std.os.linux`` and not something more common like ``std.os.unix`` (if that even exists). A benefit of putting it in ``std.c`` is that it's technically common amongst all *nix libcs.